scripts/launch-debug.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env zsh
```

Comment:
Probably safer as not everyone uses zsh

```suggestion
#!/usr/bin/env sh
```